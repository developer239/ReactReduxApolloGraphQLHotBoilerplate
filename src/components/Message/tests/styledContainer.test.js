import React from 'react'
import { shallow } from 'enzyme'
import { theme } from 'styles'
import 'jest-styled-components'
import { Container } from '../index'


describe('Container Component', () => {
  it('has correct default styles', () => {
    const wrapper = shallow(
      <Container theme={theme} />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
