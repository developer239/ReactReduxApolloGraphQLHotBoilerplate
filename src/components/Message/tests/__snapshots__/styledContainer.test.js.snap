// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component has correct default styles 1`] = `
.c0 {
  background-color: #ccd1d9;
  color: #fff;
  font-size: 14px;
  padding: 15px 20px;
}

<div
  className="c0"
/>
`;
